frontend/views/containers/GroupPledgesGraph.vue

```diff
@@ -11,17 +11,20 @@
     graph-legend-item(
       :amount='withCurrency(graphData.pledgeTotal)'
       color='primary-solid'
+      variant='side'
     ) {{ L('Total Pledged') }}
```

Comment:
@mmbotelho and @pieer

![image](https://user-images.githubusercontent.com/14869087/72247833-adf11280-35ed-11ea-9d37-4bc67d03bfc7.png)

When I read this graph, I think “total pledged” should be renamed to “total given” because "pledged" is  1375 (given) + 1196 (surplus).
Pledging isn't the same as giving, right? I may pledge 100 and only give 60. In other words, the blue label (given) would be 60 and the green (surplus) would be 40. What do you think?

cc @dotmacro and @taoeffect

Reply:
Makes a lot of sense, updated mockups on Figma!

Reply:
I agree the graph is confusing, but I don't think changing "total pledged" to "total given" is the right fix, for a few reasons. 

- "given" is independent from "surplus" because `surplus = pledge - need`... and `given = sent`. Whether or not we show "given" (i.e. payments made), it's independent from the surplus.
- For the current month's overview, "given" will change as users make payments because "given" is past tense. (This means GI will not know the final total given in a particular month until the month is over.) 
- It's important to the group to know the total pledged and the resulting surplus/need so members can adjust pledges, have conversations, etc. [It may also be important to know the total amount given. Is that more important than total pledges? Should both be shown?]